Stackless/module/taskletobject.c

```diff
@@ -361,6 +432,15 @@ PyTasklet_BindEx(PyTaskletObject *task, PyObject *func, PyObject *args, PyObject
         RUNTIME_ERROR("can't unbind the main tasklet", -1);
     }
 
+    /*
+     * Set the context to the current context. It can be changed later on.
+     * But only for non-main tasklest, because tasklet.set_context must not
+     * be used for a main tasklet.
```

Comment:
Is this explained somewhere? why it cannot be used for the main tasklet?

Reply:
Oh, the comment is no longer up to date. I'll change it.

In a earlier version of patch I thought, that setting the context of the main tasklet could cause problems.